zeq/detail/exit.fbs

```diff
@@ -0,0 +1,6 @@
+namespace zeq.vocabulary;
+
+table Exit {
+}
+
+root_type Exit;
```

Comment:
As there is no data for that, this schema had no real value, but the code generation uses that to create the according event type.